src/features/projects/components/PopupProject.tsx

```diff
@@ -37,21 +41,27 @@ export default function PopupProject({
     '%';
 
   const projectDetails = project.properties;
+
+  const currency = getStoredCurrency();
+  const country = localStorage.getItem('countryCode');
+  const language = localStorage.getItem('language');
+
+  const getSourceUrl = React.useCallback((): string => {
+    let sourceUrl = `${process.env.NEXT_PUBLIC_DONATION_URL}/?to=${projectDetails.slug}&returnToUrl=${window.location.href}&country=${country}&currency=${currency}&locale=${language}${user ? '&autoLogin=true' : ''}&tenant=${process.env.TENANTID}`;
```

Comment:
This does not yet allow to transfer the user token or the treecounter to supporter like e.g. our native Treecounter-App at https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/370706c6c03917fccec7ff30975b92dd86aafd10/app/components/PlantProjects/PlantProjectFull.native.js#L219

Reply:
Done